src/main/g8/build.sbt

```diff
@@ -1,13 +1,13 @@
 import org.scalatra.sbt._
 import org.scalatra.sbt.PluginKeys._
-import com.mojolly.scalate.ScalatePlugin._
+import ScalateKeys._
 import ScalateKeys._
 
 val ScalatraVersion = "$scalatra_version$"
 
 ScalatraPlugin.scalatraSettings
 
-scalateSettings
+seq(scalateSettings:_*)
```

Comment:
```
[error] /tmp/sbt_deb7d41e/scripted/build.sbt:10: warning: method seq in trait BuildExtra is deprecated: In build.sbt files, this call can be removed.  In other cases, this can usually be replaced by Seq.
[error] seq(scalateSettings:_*)
[error] ^
```